homeassistant/helpers/system_info.py

```diff
@@ -41,8 +41,11 @@ async def async_get_system_info(hass: HomeAssistant) -> dict[str, Any]:
 
     # Determine installation type on current data
     if info_object["docker"]:
-        if info_object["user"] == "root":
+        if os.path.isfile("/OFFICIAL_IMAGE"):
```

Comment:
We should not replace this, we can add, but "root" is also required

Reply:
fixed